data/183/012/397/1/1830123971.geojson

```diff
@@ -60,10 +66,10 @@
[83.479383,27.559702],[83.479387,27.559882],[83.479286,27.559884],[83.479292,27.560154],[83.479191,27.560156],[83.479193,27.560246],[83.479091,27.560248],[83.47909300000001,27.560338],[83.478891,27.560342],[83.478893,27.560432],[83.478792,27.560434],[83.47879399999999,27.560524],[83.47869300000001,27.560526],[83.47869900000001,27.560811],[83.47870500000001,27.561067],[83.47880600000001,27.561065],[83.478812,27.561336],[83.47891300000001,27.561334],[83.478917,27.561515],[83.47901899999999,27.561513],[83.479023,27.561693],[83.479124,27.561691],[83.479128,27.561872],[83.479229,27.56187],[83.479231,27.56196],[83.479332,27.561958],[83.47933399999999,27.562048],[83.47943600000001,27.562047],[83.479438,27.562137],[83.479539,27.562135],[83.479541,27.562225],[83.479642,27.562223],[83.47964399999999,27.562314],[83.47954300000001,27.562315],[83.479553,27.562766],[83.47945199999999,27.562768],[83.479454,27.562858],[83.479353,27.56286],[83.47935699999999,27.563041],[83.47925499999999,27.563042],[83.479264,27.563403],[83.479162,27.563405],[83.479179,27.564127],[83.479077,27.564128],[83.479079,27.564219],[83.478978,27.56422],[83.47898000000001,27.564311],[83.47857500000001,27.564318],[83.478573,27.564228],[83.478472,27.564229],[83.47847,27.564139],[83.478369,27.564141],[83.47836700000001,27.564051],[83.47816400000001,27.564054],[83.478162,27.563964],[83.477555,27.563975],[83.477553,27.563885],[83.47725,27.56389],[83.477248,27.5638],[83.477146,27.563802],[83.477144,27.563712],[83.47694199999999,27.563715],[83.47694,27.563625],[83.476839,27.563627],[83.476837,27.563537],[83.476634,27.56354],[83.476632,27.56345],[83.47642999999999,27.563454],[83.476428,27.563364],[83.476225,27.563367],[83.476223,27.563277],[83.476122,27.563279],[83.4761,27.562287],[83.475999,27.562288],[83.475995,27.562108],[83.475893,27.56211],[83.475891,27.56202],[83.47579,27.562021],[83.47578799999999,27.561931],[83.47568699999999,27.561933],[83.475685,27.561843],[83.475483,27.561847],[83.47548,27.561756],[83.475379,27.561758],[83.47537699999999,27.561668],[83.47517499999999,27.561672],[83.475173,27.561581],[83.475072,27.561583],[83.47507,27.561493],[83.474968,27.561495],[83.47496599999999,27.561405],[83.47486499999999,27.561406],[83.474863,27.561316],[83.474256,27.561327],[83.47425800000001,27.561417],[83.474056,27.561421],[83.474058,27.561511],[83.473956,27.561513],[83.473958,27.561603],[83.47375599999999,27.561607],[83.47376,27.561787],[83.473659,27.561789],[83.473669,27.56224],[83.47377,27.562238],[83.473772,27.562328],[83.473975,27.562325],[83.473977,27.562415],[83.47417900000001,27.562411],[83.474181,27.562501],[83.474282,27.5625],[83.474284,27.56259],[83.474385,27.562588],[83.474389,27.562768],[83.474491,27.562766],[83.474493,27.562857],[83.474594,27.562855],[83.47459600000001,27.562945],[83.47469700000001,27.562943],[83.474699,27.563033],[83.4748,27.563032],[83.474802,27.563122],[83.474904,27.56312],[83.474908,27.5633],[83.475009,27.563299],[83.47501099999999,27.563389],[83.475112,27.56
```

Comment:
Yeah.. unfortunately the Karmashapes geometry for this feature spans over a large portion of Nepal. I'm open to other options here, but figured since the default geometry is being stored as an alt and the new point is accurate, the ID would stay as is for this feature.. but curious what you think about this one.

Reply:
If the Karmashapes feature should have just been in India and is now just in India, I think I'm fine keeping the WOF ID the same this time as it's more like a bug fix.